backend/triplestore/rdf_model.py

```diff
@@ -0,0 +1,90 @@
+'''
+Base class for Django-esque management of RDF data.
+'''
+
+from rdflib import URIRef, Graph, RDF
+from typing import Dict, Optional
+from django.conf import settings
+
+from triplestore.rdf_field import RDFField
+from triplestore.utils import triples_to_quads, Triples
+
+class RDFModel():
+    '''
+    Abstract class for RDF data models
+
+    Create subclasses to model a data class.
```

Comment:
Also indicate how to do this: by optionally setting `rdf_class` and adding `RDFProperty`s.